verify.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+set -e
+
+CLI="faas-cli"
+func_name="validate"
```

Comment:
My suggestion would be to build with the name of the function being unique and based upon the folder name - i.e. `python3-ci` / `go-ci`, only delete from the filesystem at the very end, if there was no error detected.